### ISSUE TITLE
Any plans to normalize the database?

### ISSUE THREAD

**Issue by jorainer** [MassBank/MassBank-web#266]
I was just wondering if there are any plans to *normalize* the database, i.e. to reduce the redundancy in the `COMPOUND` table @sneumann @tsufz @meier-rene ? As far as I've seen there is one entry in the `COMPOUND` table for each entry in the `RECORD` table (i.e. the spectra).

**Comment by meier-rene** [comment]
You are right. Every RECORD creates a entry in COMPOUND and this is a conceptional mistake but it has very low priority to fix this on my side. It has somehow historical origin which I dont know.

**Comment by tsufz** [comment]
Yes, I think this an back log issues from previous development. The compound name is not yet standardised and thus each records maybe unique in terms of "any name". We were already thinking about a preferred MassBank name which is then shown in the records and the website. However as @meier-rene said, this has not too high priority because is needs some (painful) curation. I have it somewhere down on my to do list, but @schymane has maybe also an opinion. We may have an issue open...
